src/style.css

```diff
@@ -0,0 +1,162 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato&display=swap');
+
+body {
+  /* stylelint-disable-next-line font-family-no-missing-generic-family-keyword */
```

Comment:
please do not disable eslint to ensure best practices of JavaScript

Reply:
good reviewer ever thank you 👍